javascript/ql/src/Performance/ReDoS.ql

```diff
@@ -777,6 +783,7 @@ predicate isPumpable(State fork, string w) {
  */
 State process(State fork, string w, int i) {
   isPumpable(fork, w) and
+  min(string s | isPumpable(fork, s)).prefix(w.length()) = w and
```

Comment:
That one got removed later, exactly because of the reasons you mention. (See the change in the `select`, where I moved some `not` out of the `min(..)`).    

We did in fact only look at the minimum pumpable string, so we did not loose any results when this change was made.